src/bio/bio_monitor.c

```diff
@@ -177,23 +238,12 @@ bio_dev_set_faulty(struct bio_xs_context *xs)
 	return rc;
 }
 
-static inline struct bio_dev_health *
-xs_ctxt2dev_health(struct bio_xs_context *ctxt)
-{
-	D_ASSERT(ctxt != NULL);
-	/* bio_xsctxt_free() is underway */
-	if (ctxt->bxc_blobstore == NULL)
-		return NULL;
```

Comment:
Where is the equivalent logic in new code?

Reply:
before we passed bio_xs_context as arg, and use this helper to get bio_dev_health, now we passed bio_blobstore as arg directly.